src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -77,6 +77,12 @@ public CollectionExerciseEndpoint(
   public CollectionExerciseDto getCollex(
       @PathVariable(value = "collexId") UUID collexId,
       @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    authUser.checkUserPermission(
+            userEmail,
+            collectionExercise.getSurvey().getId(),
+            UserGroupAuthorisedActivityType.VIEW_COLLECTION_EXERCISE);
```

Comment:
I don't think this will work here, it needs to fetch the collection exercise first to know the survey ID it's checking.